frontend/src/features/SideWindow/MissionForm/ActionList/Item.tsx

```diff
@@ -1,73 +1,103 @@
-import { Accent, Icon, IconButton, Size, THEME } from '@mtes-mct/monitor-ui'
+import { Accent, getLocalizedDayjs, Icon, IconButton, Size, THEME } from '@mtes-mct/monitor-ui'
 import { useMemo } from 'react'
 import styled from 'styled-components'
 
 import { formatDateLabel } from './utils'
-import { MissionType } from '../../../../domain/types/mission'
-import { getLocalizedDayjs } from '../../../../utils/getLocalizedDayjs'
+import { MissionAction } from '../../../../domain/types/missionAction'
+import { FrontendError } from '../../../../libs/FrontendError'
 
-import type { Action, PartialAction } from '../types'
+import type { MissionActionFormValues } from '../types'
 import type { Promisable } from 'type-fest'
 
-export type ItemProps = (
-  | {
-      action: PartialAction
-      isNew: true
-    }
-  | {
-      action: Action
-      isNew?: false
-    }
-) & {
+const ERROR_PATH = 'features/SideWindow/MissionForm/ActionList/Item.tsx'
+
+export type ItemProps = {
+  initialValues: MissionActionFormValues
+  isSelected: boolean
   onDelete: () => Promisable<void>
+  onDuplicate: () => Promisable<void>
+  onEdit: () => Promisable<void>
 }
-export function Item({ action, isNew = false, onDelete }: ItemProps) {
+export function Item({ initialValues, isSelected, onDelete, onDuplicate, onEdit }: ItemProps) {
   const [actionLabel, ActionIcon] = useMemo(() => {
-    switch (isNew) {
+    switch (initialValues.isDraft) {
```

Comment:
Why show this text in draft mode ? I think the `false` case text of the switch should be showed if there is at least one form field not null

Reply:
There : https://xd.adobe.com/view/0eacf501-00f4-496b-bf08-4e9ca7985909-5bff/screen/3e67be9f-984b-43ab-9f8b-28e0cc23e8c0/
"Contrôle en mer à renseigner" is no more showed when there is a value.